### PR TITLE
Remove pocs.utils.signal-to-noise in favour of gungala

### PR DESCRIPTION
**Pull request by AnthonyHorton**
Removes the old code in `pocs.utils.signal-to-noise` and replaces it with the standalone [gunagala](https://github.com/AstroHuntsman/gunagala) SNR/ETC/sensitivity library, which was based on a combination of code from `poc.utils.signal-to-noise` and the Australian Space Eye performance models, and has since been improved and extended. 

 When merged this will close Issue #31.

### CODE REVIEW

**Comment by AnthonyHorton** [comment]
Test failures are due to intermittent bugs in `POCS` messaging that have been plaguing the `AstroHuntsman` fork of `POCS` for some time (yet another reason for rebaselining `AstroHuntsman/POCS` off `panoptes/POCS` version 1.0). The tests on the code relevant to this PR are passing.

File: pocs/tests/test_hdr.py [hunk 1]
```diff
@@ -1,51 +1,70 @@
...
-                                      base_position=base)
+        for filter_name in imagers[imager_name].filters:
+            exposure_parameters = {'filter_name': filter_name,
+                                   'shortest_exp_time': 5 * u.second,
+                                   'longest_exp_time': 600 * u.second,
+                                   'num_long_exp': 1,
+                                   'exp_time_ratio': 2.0,
+                                   'snr_target': 5.0}
+            targets = hdr.get_target_list(target_name=name,
+                                          imagers=imagers,
```
Comment (wtgee) [inline pocs/tests/test_hdr.py]: Are all the `imagers` still required if looping over the filters here?

**Comment by wtgee** [comment]
I want to start putting more notebooks in for the examples and think we might get cluttered soon. Can we maybe move to a subdir? Something like `examples/notebooks/gunagla/` (or whatever might be most appropriate)

**Comment by AnthonyHorton** [comment]
Subdirectories for notebooks is a good idea.

File: pocs/tests/test_hdr.py (hunk 1 above)
Comment (AnthonyHorton) [reply pocs/tests/test_hdr.py]: In `gunagala` they are separate things, each `Imager` has a dictionary of available filters. Unlike the old `signal-to-noise` code we don't have to have a separate `Imager` instance for each filter.

This code is iterating over the 4 imagers defined in `gunagala`'s default config files, which are 5 lens Huntsman arrays for dark time, the same for bright time, and 2 different versions of the Australian Space Eye concept with different filter sets. Really shouldn't have to have separate `Imager` instances just for different phases of the Moon, either...

**Pull request merged**